README.md

```diff
@@ -125,42 +125,24 @@ NOTE: The url of the jupyter server will be displayed in the docker-compose outp
 (default: `http://127.0.0.1:8888/?token=SOME_JUPYTER_TOKEN`) (you can customize jupyter port and host using [env](#env-variables) variables)
 
 ### Training
-Currently GPU support is available only in the experimental jupyter notebook `neural_cf` that contains whole ML cycle:
-- Datasets creation, preprocessing and inspections
-- Models creation (Autoencoders, NCF)
-- Trainings
-- Evaluation
-- Inferention
 
-GPU support can be enabled using environmental variable TRAINING_DEVICE (look into [ENV variables](#env-variables) section) , but for now it doesn't work in the dev/test/prod environments due to the fact that celery uses `fork` rather than `spawn` multiprocessing method - it is incompatibile with `CUDA`.
+Recommender system can use one of two recommendation engines implemented as agents:
+- `Pre Agent` - that implements [Neural Collaborative Filtering](https://arxiv.org/abs/1708.05031) algorithm.
+- `RL Agent` - that implements [Deep Deterministic Policy Gradient](https://arxiv.org/abs/1509.02971) algorithm.
 
-For now the only trainable recommendation agent is the Pre-Agent.
+To choose prefered agent, use `AGENT_VERSION` env variable (look into [ENV variables](#env-variables) section). After change of this variable in the .env file no reloading is needed - system will use appropriate agent dynamically (if it is trained).
```

Comment:
Use `AGENT_VERSION` env variable to select an agent version (look into [ENV variables](#env-variables) section). You don't have to restart the server after the modification of this variable - the system will use an appropriate agent dynamically (if it is trained).